_includes/setup/path-win.md

```diff
@@ -0,0 +1,23 @@
+### Update your path
```

Comment:
Do we need these instructions? `setx` mentioned above should already be permanent (at least that's my understanding after reading https://ss64.com/nt/setx.html). No need to go through the UI.